app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable shared among all instances of `Animal`. This can lead to issues in a multithreaded environment where multiple threads might modify the list simultaneously. Consider using a thread-safe data structure or implementing proper synchronization if this is a concern.